.travis.yml

```diff
@@ -1,13 +1,33 @@
 sudo: false
 language: node_js
-script:
-  - npm run build
-  - npm run test:ci
 node_js:
+  - 6
   - 8
   - 10
   - 12
 
+before_install:
+  - npm config set depth 0
+before_script: >
```

Comment:
Would it be easier to use the `matrix` feature that we use below for switching on node version instead of this?